R/transformCounts.R

```diff
@@ -13,7 +13,8 @@
 #'   (Default: \code{"samples"})
 #' 
 #' @param pseudocount \code{Logical scalar} or \code{numeric scalar}. When TRUE,
-#'   automatically adds half of the minimum positive value of \code{assay.type}.
+#'   missing values(NAs) are ignored and, it automatically adds half of the 
+#'   minimum positive value of \code{assay.type}.
```

Comment:
Missing values is just a side note. Better description might be

When \code{TRUE}, automatically adds half of the minimum positive value of \code{assay.type} (missing values ignored by default: \code{na.rm = TRUE}).

Reply:
Thanks